### PR TITLE
Use infix version style for applying plugins

### PR DESCRIPTION
**Pull request by Goooler**
Easy to use in both Groovy and Kotlin DSL

### CODE REVIEW

**Comment by martinbonnin** [comment]
I think both work in Groovy and Kotlin?

File: intellij-plugin/src/test/testData/migration/v2-to-v3/upgradeGradlePluginInBuildGradleKts.gradle.kts [hunk 1]
```diff
@@ -2,11 +2,11 @@ plugins {
...
   kotlin("jvm") version "1.6.10"
   id("com.apollographql.apollo")
-  id("com.apollographql.apollo").version("2.5.14")
-  id("com.apollographql.apollo").version("2.5.14").apply(false)
-  id("com.apollographql.apollo").version("2.5.14") apply (false)
-  id("com.apollographql.apollo").version(someClass.someConstant)
-  id("com.apollographql.apollo").version("${someClass.someConstant}")
+  id("com.apollographql.apollo") version "2.5.14"
+  id("com.apollographql.apollo") version "2.5.14" apply false
+  id("com.apollographql.apollo") version "2.5.14" apply false
```
Comment (Goooler) [inline intellij-plugin/src/test/testData/migration/v2-to-v3/upgradeGradlePluginInBuildGradleKts.gradle.kts:7]: Redundant?

File: intellij-plugin/src/test/testData/migration/v2-to-v3/upgradeGradlePluginInBuildGradleKts_after.gradle.kts [hunk 2]
```diff
@@ -2,13 +2,13 @@ plugins {
   java
   kotlin("jvm") version "1.6.10"
   id("com.apollographql.apollo3")
-  id("com.apollographql.apollo3").version("3.7.4")
-  id("com.apollographql.apollo3").version("3.7.4").apply(false)
-  id("com.apollographql.apollo3").version("3.7.4") apply (false)
+  id("com.apollographql.apollo3") version "3.7.4"
+  id("com.apollographql.apollo3") version "3.7.4" apply false
+  id("com.apollographql.apollo3") version "3.7.4" apply false
```
Comment (Goooler) [inline intellij-plugin/src/test/testData/migration/v2-to-v3/upgradeGradlePluginInBuildGradleKts_after.gradle.kts:7]: As below.

**Comment by martinbonnin** [comment]
>> I think both work in Groovy and Kotlin?
> Yep.

What I mean is: we don't need infix notations in Groovy. Is this PR about code style + deduplications?

**Comment by Goooler** [comment]
Yea, but that's the more idiomatic way.

**Comment by martinbonnin** [comment]
ok

**Review by martinbonnin — APPROVED** [review]

**Pull request merged**